karyon3-core/src/main/java/com/netflix/karyon/KaryonFeatures.java

```diff
@@ -13,6 +13,11 @@
      */
     public final static KaryonFeature<Boolean> SHUTDOWN_ON_ERROR = KaryonFeature.create("karyon.features.shutdownOnError", true);
     
+    /**
+     * Auto discover AutoBinders using the ServiceLoader
+     */
+    public static final KaryonFeature<Boolean> DISCOVER_AUTO_BINDERS = KaryonFeature.create("karyon.features.discoverAutoBinders", true);
```

Comment:
Doesnt' match the format of surrounding items should be "public final static" though I prefer "public static final" as you have it here, so maybe the surrounding items should change.  Also, can we make these package private?  It doesn't seem that they need to be exposed since there is a separate mechanism for the user to override these values and these are merely the fallbacks if the user specifies nothing.